scalpel/src/Text/HTML/Scalpel.hs

```diff
@@ -157,7 +157,11 @@ module Text.HTML.Scalpel (
 ,   seekBack
 ,   untilNext
 ,   untilBack
+
+-- * Select
+, select
```

Comment:
What is your use case for exposing select?

I'd prefer to keep it hidden since it gives me more flexibility to make changes to the internals of library. It's type has already changed several times over the life of the library.

Reply:
You are right. I was thinking I would need it but was able to accomplish what I wanted with functors.